Bottom: ec9bbe24a4ece15eff809faac4af300b1976af4e
Top:    03015ebc183da5cad6173f595426af904cfe88eb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:13:04 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c895f19..5c7b9f9 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -755,7 +755,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -768,7 +768,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev)
     {
       glVertex2i (0,                  y_prev - VCD);
